ElectronClient/app/gui/NoteText.jsx

```diff
@@ -880,9 +880,93 @@ class NoteTextComponent extends React.Component {
 				}
 				return output;
 			}
-			
-			//fixes #1426 but this is an Ace issue, so it can be removed if ace/brace is updated.
+
+			// fixes #1426 but this is an Ace issue, so it can be removed if ace/brace is updated.
+			// START of very ugly hack... (but it works)
+			var getWrapped = function(selection, selected, opening, closing) {
```

Comment:
What is this getWrapped function for? It doesn't seem to be used so maybe we can remove it?

Reply:
Edit: sorry it does get used.